compiler/noirc_frontend/src/hir/comptime/hir_to_display_ast.rs

```diff
@@ -212,6 +212,17 @@ impl HirExpression {
 
             // A macro was evaluated here: return the quoted result
             HirExpression::Unquote(block) => ExpressionKind::Quote(block.clone()),
+
+            // Convert this back into a function call `Enum::Foo(args)`
+            HirExpression::EnumConstructor(constructor) => {
+                let typ = constructor.r#type.borrow();
+                let variant = &typ.variant_at(constructor.variant_index);
+                let path = Path::from_single(variant.name.to_string(), span);
```

Comment:
Should this path have both segments? Or it doesn't matter?

Reply:
Added